tests/configs/test-006-ch-upgrade-2.yaml

```diff
@@ -11,7 +11,7 @@ spec:
       spec:
         containers:
         - name: clickhouse-pod
-          image: yandex/clickhouse-server:21.1.7.1
+          image: altinity/clickhouse-server:21.8.altinity_prestable
```

Comment:
please rollback this changes upgrade-2 shall contains old version, maybe 21.3 for example